databases/tests/conftest.py

```diff
@@ -0,0 +1,101 @@
+import os
+import asyncio
+import inspect
+from typing import TYPE_CHECKING, Iterator
+
+import pytest
+
+import prisma
+from prisma import Prisma
+from prisma.testing import reset_client
+from prisma.utils import get_or_create_event_loop
+
+from lib.testing import async_fixture
+from .utils import RAW_QUERIES_MAPPING, RawQueries
+from ..utils import DatabaseConfig
+
+
+if TYPE_CHECKING:
+    from _pytest.fixtures import FixtureRequest
+
+
+# TODO: a lot of this is copied from `tests/conftest.py`, figure out a good way to share
```

Comment:
Add this to the TODO tracker. Is it as simple as imports from conftest? It might be a little bit an an anti-pattern but I think the value of code re-use trumps

Reply:
Good idea, I think that sort of thing is a bit weird but works really well for this case, done in cbddf0b6f2070c48e9fdd5b158b4dcb2fdbd3f2e